src/repos/questbook-analytics-v0.ts

```diff
@@ -0,0 +1,56 @@
+import { writeFile } from "fs/promises";
+import { join } from "path";
+import { CIRepo } from "../types";
+
+const CHAINS_DATA_FILE = './src/psuedo-generated/chainInfo.json'
+
+const questbookAnalyticsV0: CIRepo = {
+	repoName: 'questbook-analytics-v0',
+	doCI: async(repoPath, chains) => {
+		// generate the chains.json file
+		const data = Object.values(chains).reduce(
```

Comment:
This part of the code from line number 11 to 47 is being used for other repos (qb-api, etc) as well, can we move this to a common function under a file in utils (create a new file) and use it wherever required? @richikchanda1999